### ISSUE TITLE
A CALL TO ORDER

### ISSUE THREAD

**Issue by realcodywburns** [ethereumproject/project-funding#3]
Ethereum<sup>Classic</sup> Voting Proposal System.

I have pinged you all to propose a call for a vote on how to handle funds which are donated to the dev team. In this repository you will find the basic framework of the system I think would work. The goal is to provide transparency to allow investors to feel safer donating to the cause as well as fund projects which will be capital intensive. 

If you are in agreement, add yourself on the table in the sig file. 

To be clear: this is not a call to form a foundation or any sort of governing body of the etc. 

@arvicco  @cjdelisle @bart-w0 @cjdelisle @dustysesameseed @elaineo @igetgames @jeffanthony @JikkuJose @jsmith-dev @Machete3000 @marcusyh @matthewjamesr @MrComputerCodes @nnadir @ProphetDaniel @splix @vodnic @whatisgravity

**Comment by elaineo** [comment]
Does this mean there will be a president? (I'm not against the idea, just clarifying)

Also, based on the contract code, it looks like minimum quorum and debating period are set upon creation, although I would expect that they would be unique to each proposal (big proposals may need higher bar for quorum, longer debating period). Other than that I like the general idea.

**Comment by realcodywburns** [comment]
Yes a 'president' would be chosen to initiate the contract as it presently is written.

**Comment by realcodywburns** [comment]
The reason for a 'president' of the board is that this will not be an decentralized autonomous  organisation (DAO). It will be decentralized. It will be an organisation. But history has shown that entrusting money to an autonomous code may not be the best idea when the parties are anonymous....

**Comment by matthewjamesr** [comment]
@realcodywburns I agree with this statement and am reading the files now. I propose that all signatures be pgp-signed.

**Comment by matthewjamesr** [comment]
All, I created a pgp proof list at the file below. Please follow the format I used to include yourselves.

[PGP Proof List](https://github.com/ethereumproject/project-funding/blob/master/developers_pgp_proof.md)

**Comment by whatisgravity** [comment]
I'm putting in my vote for a x-of-x multisig system with very limited output functions.

**Comment by whatisgravity** [comment]
in my opinion, regardless of how we should raise money for development, or even regardless how we propose to distribute it to developers, we need a secure way to handle basic storage in a transparent, secure and decentralized way.

I do think it needs to be more complex than just the multisig wallet. In my opinion we should focus on that goal, then we can propose modular updates to that and discuss and refine which interact with the neutral base. It also lets us to make progress while allowing us to put off major decisions, which will allow more people to provide their input.

I believe the most pressing issues are:

    How do we decide who holds the original multisig keys.
    How will the output functions work
    What is the threshold needed to generate an output (per output type?)
    Determine if the voting system should be in this contract or decide what API will it need to interface with a voting module.

I believe everyone can agree with these basic functional requirements regardless on how they feel the specifics of the system. If these are in fact the common features to every system, I believe this is the best starting point.

If you disagree on this path, let me know what issues you see or what alternative you think would be better

(copied from the per project vs organization funding thread)

**Comment by JikkuJose** [comment]
I agree to @whatisgravity just mentioned. Specifially:

>  we need a secure way to handle basic storage in a transparent, secure and decentralized way.

This is very fundamental. Irrespective of what each one in the community feels towards the future, we need to arrive at a consensus on securing the funds.

**Comment by ProphetDaniel** [comment]
Created consider.it site ([see Github issue](https://github.com/ethereumproject/project-funding/issues/5)) for auxiliating the decision taking process in the organization with transparency.
More information at [etc.consider.it](https://etc.consider.it/).

**Comment by avtarsehra** [comment]
I know this is a bit off topic, but may be relevant due to the direction the conversation is taking:  Has anyone considered forking the Open Bazzar (OB) project to utilise as a decentralised mechanism for posting and funding projects? I am not familiar with the finer details of OB, but I think it would be fairly straight forward to adapt to project funding, and the moderated payment process can be leveraged to collate funds until a threshold is reached or a time expiry is reached. Unlike in the Bitcoin OB this ETC funding marketplace moderated process can be a very basic smart contract, but it could have some form of multisig release to avoid funds getting stuck. This is only an idea. As I mention I am not that familiar with the OB project, and there may be complexities which mean this would be challenging. Or it could be layer that is added at a later date... Thoughts?

**Comment by ProphetDaniel** [comment]
@avtarsehra, Open Bazzar philosophy is inline with our manifest. It looks a great way for presenting the projects/ideas to the viewers. Specializing it for our cause is needed though.

**Comment by avtarsehra** [comment]
@ProphetDaniel unfortunately I have a huge amount on my plate at the moment, but if someone wants to initiate the OB type fork project, I would be fully on board. Do we have any takers? :-)

**Comment by whatisgravity** [comment]
I think we would need to discuss the specifics more, I would be happy to see a mockup but the fundamentals of the system still need to be discussed.

**Comment by avtarsehra** [comment]
@whatisgravity yes definitely - my overly curious nature takes over sometimes! 

I've downloaded the OB client and running it now - it's actually a very good piece of work! I will try and get a little more information on it to share with everyone for further discussion.

**Comment by ProphetDaniel** [comment]
@whatisgravity @avtarsehra , an specific [issue](https://github.com/ethereumproject/project-funding/issues/6) was opened to address the last comments from you.

**Comment by avtarsehra** [comment]
@Prophetdaniel thank you I will upload my analysis there when done.
